### PR TITLE
add single video page layout

### PR DESCRIPTION
**Pull request by hsnice16**
image

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -16,44 +16,60 @@
   --NAVBAR-Z-INDEX: 3;
   --CATEGORY-CHIP-CONTAINER-Z-INDEX: 2;
   --VIDEO-CARD-DOTS-ICON-Z-INDEX: 1;
```
Comment (vrx29) [inline src/App.css:18]: Nice work you have used variables to keep a track of Z-index

**Review by vrx29 — APPROVED** [review]
Your code is clean and well written. It's hard to find mistakes or suggest improvements. You can go ahead.

**Review by foolhardy21 — APPROVED** [review]
like the `SideNavBar` component. code organization is good too. nice work.

**Comment by hsnice16** [comment]
Thank you for reviewing,

**Pull request merged**